app/views/planning_applications/validation_decision.html.erb

```diff
@@ -38,17 +38,14 @@
           <%= "The application is marked as invalid. The applicant was notified on #{@planning_application.invalidated_at}" %>
         </strong>
       </p>
-      <p class="govuk-body">
-        <%= link_to "Check documents", planning_application_documents_path(@planning_application), class: "govuk-link" %>
-      </p>
```

Comment:
Is removing this intentional? I couldn't see it in the ACs but there's a lot going on so fine if it is 😅

Reply:
Going off the design prototypes and I think I cleared it with Tom.
I'll double check for sure, thanks for asking!